content/javascript/concepts/strings/terms/valueOf/valueOf.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.valueOf()'
+Description: 'Returns the value of a String object as a string.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Objects'
+  - 'Strings'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.valueOf()`** method returns the [primitive](https://www.codecademy.com/resources/docs/javascript/data-types) value of a `String` object, similar to the `.toString()` method of JavaScript Strings. Usually this method isn't used in code but is called internally by JavaScript.
+
+> **Note:** This `String` method is not to be confused with the `.valueOf()` method that exists for the `Object` class.
```

Comment:
Can we use a different perspective for this note and say something like:

"Objects that are descendants of the `Object` class, including `String`s, inherit the `valueOf()` method."